recipes/d3ploy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "d3ploy" %}
+{% set version = "0.8" %}
+{% set sha256 = "f98c4984974a834963b474be394f10aac718d72ba5ea48c0da2ca098ff202943" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
this should probably be changed to `name: {{ name|lower }}` because [d3ploy](https://d3ploy.readthedocs.io/en/latest/index.html) is a different, more well established Python project :(

Reply:
edit: `name: cyclus-{{ name|lower }}`